app/locales/en/translations.js

```diff
@@ -0,0 +1,233 @@
+export default {
```

Comment:
The [ember-i18n wiki covers this](/jamesarosen/ember-i18n/wiki/Doc:-Defining-Translations#location-fallback)--see location fallback and region subtags